notes.md

```diff
@@ -0,0 +1,18 @@
+difficulties faced in heroku :
+	creating password
+	creating app name
```

Comment:
Can you explain clearly what are issues you faced when creating password and app name